From: David Paleino <dapal@debian.org>
Subject: correctly support LFS
Origin: vendor
Forwarded: no

---
 configure.ac |    2 ++
 1 file changed, 2 insertions(+)

--- a/configure.ac
+++ b/configure.ac
@@ -21,6 +21,8 @@ AM_PROG_LIBTOOL
 
 AM_SILENT_RULES([yes])
 
+AC_SYS_LARGEFILE
+
 # Library dependencies
 PKG_CHECK_MODULES(GLIB,     [glib-2.0 >= 2.16])
 PKG_CHECK_MODULES(GTK,      [gtk+-3.0 >= 3.0])
